### ISSUE TITLE
Some questions about Prior-Association

### ISSUE THREAD

**Issue by A-New-Page** [thuml/Anomaly-Transformer#22]
Thanks for submitting such a great job, but I have some questions with prior-association of the code

https://github.com/thuml/Anomaly-Transformer/blob/72a71e5f0847bd14ba0253de899f7b0d5ba6ee97/model/attn.py#L48-L49

In this part, some mathematical processing is done on the `sigma`
Could you please explain some reasons for doing this?
 I can not find any reference in your paper.

**Comment by wuhaixu2016** [comment]
This is a trick to make the training more stable. 
In our previous experiments, if you just use the sigma directly, you can also obtain a good performance.

**Comment by A-New-Page** [comment]
Thank you for your reply.I have another question about the dataset.

In the experimental dataset, are there any abnormal point in the training set?
In appendix K of this paper, Table 13 presents the information about experimental dataset, where AR represents the truth abnormal proportion of the whole dataset. Does AR represent the abnormal proportion in the test set?

**Comment by wuhaixu2016** [comment]
(1) The training set does contain abnormal points. Since we focus on the unsupervised method, we do not use the label in the training process.
(2) AR is counted from the combination of all data, including both the training and test.

**Comment by A-New-Page** [comment]
Thanks!
